.github/workflows/verify.yml

```diff
@@ -29,6 +29,19 @@ jobs:
         with:
           version: ${{ env.GOLANGCI_LINT_VERSION }}
 
+  lint-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Lint the markdown
+        uses: DavidAnson/markdownlint-cli2-action@v10
+        with:
+          globs: |
+            **/*.md
+            !vendor
+            !docs
```

Comment:
I'm not sure if excluding the docs folder is the best idea. But I only saw auto generated stuff there..

Reply:
yeah this works for me for now. it's possible in the future we may want the docs we generate to be linted (i.e. fix the generator) but I'm fine with that being out of scope here

Reply:
I triggered the actions, and the [lint-markdown job](https://github.com/cri-o/cri-o/actions/runs/4950189109/jobs/8875431902?pr=6890) passed 🎉 . 

Wondering if this job should be required, meaning it blocks the PR merge in case of a failure.  @haircommander  WDYT?

Reply:
This is the command I used to test everything myself:

```shell
npx markdownlint-cli2 "**/*.md" "\!vendor" "\!docs" "\!.github"
```
`npm install -g` is also a possibility but I like [npx](https://docs.npmjs.com/cli/v7/commands/npx)

I'm not really experienced in make files. Is this something that can be included in some way to make formatting easier?

Reply:
> Wondering if this job should be required, meaning it blocks the PR merge in case of a failure. @haircommander WDYT?

I'm not sure we can block on a gh action (at least from prow's perspective

> I'm not really experienced in make files. Is this something that can be included in some way to make formatting easier?

yeah in a similar to our targets like `make lint` we could have a job that runs this

Reply:
Makes sense.